backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -51,6 +51,9 @@ private val threadLocalAnnotationFqName = FqName("kotlin.native.ThreadLocal")
 val IrClassSymbol.objectIsShared get() =
     !descriptor.annotations.hasAnnotation(threadLocalAnnotationFqName)
 
+val IrFieldSymbol.isShared get() =
```

Comment:
We should avoid using `Ir*Symbol` classes in signatures and use IR declaration classes instead.